tests/requirements.txt

```diff
@@ -34,3 +33,4 @@ pytest-xdist
 requests
 responses
 ruamel_yaml
+toolz
```

Comment:
```suggestion
# conda-build to drop cytoolz dependency (https://github.com/conda/conda-build/pull/4556)
# once that occurs this can be relaxed to toolz (https://github.com/conda/conda/pull/11700)
cytoolz
```

Reply:
No longer necessary with https://github.com/conda/conda/pull/11704